public/app/features/explore/AddToDashboard/__test__/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { act, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as dashboardApi from 'app/features/manage-dashboards/state/actions';
+import { DashboardSearchHit, DashboardSearchItemType } from 'app/features/search/types';
+
+const createFolder = (title: string, id: number): DashboardSearchHit => ({
+  title,
+  id,
+  isStarred: false,
+  type: DashboardSearchItemType.DashFolder,
+  items: [],
+  url: '',
+  uri: '',
+  tags: [],
+});
+
+const foldersSearchPromise = Promise.resolve([createFolder('Folder 1', 0), createFolder('Folder 2', 0)]);
+jest.spyOn(dashboardApi, 'searchFolders').mockReturnValue(foldersSearchPromise);
+
+export const openModal = async () => {
```

Comment:
Is it being used anywhere? I don't see any direct imports 🤔

Reply:
good point, it's a leftover file from previous tests, remove in 9d481c1449